### PR TITLE
Sg2 13 insurance policy - after broken branches

### PR DESCRIPTION
**Pull request by Jake-AS**

### CODE REVIEW

File: app/validators/insurance_policy.ts [hunk 1]
```diff
@@ -0,0 +1,78 @@
+import { TableName } from 'app/enums';
+import { InsurancePolicy } from '../models';
+import Validator, { Type } from './validator';
+class InsurancePolicyValidator extends Validator<InsurancePolicy> {
```
Comment (deanylev) [inline app/validators/insurance_policy.ts:4]: Missing a newline before the class

File: app/validators/insurance_policy.ts [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+    constructor() {
+        super();
```
Comment (deanylev) [inline app/validators/insurance_policy.ts:6]: This file is still using 4-space indentation instead of 2

File: app/input-validator.ts [hunk 3]
```diff
@@ -4,12 +4,14 @@ import { isPlainObject, startCase } from 'lodash';
 import database from './database';
 import { TableName } from './enums';
 import {
-  Employee
+  Employee,
+  insurance_policy,
```
Comment (deanylev) [inline app/input-validator.ts:8]: This needs to be `InsurancePolicy` now, same on line 134

Comment (Jake-AS) [reply app/input-validator.ts:8]: fixed.

File: app/validators/insurance_policy.ts (hunk 1 above)
Comment (Jake-AS) [reply app/validators/insurance_policy.ts:4]: fixed.

**Pull request merged**